.changeset/selfish-actors-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
```

Comment:
Think this should be a minor change as we've added something to the api surface.

Reply:
You are correct 👍